src/scraper/scraper.cpp

```diff
@@ -862,12 +867,15 @@ void Scraper(bool bSingleShot)
 
             // If team filtering is set by policy then pull down and retrieve team ID's as needed. This loads the TeamIDMap global.
             // Note that the call(s) to ScraperDirectoryAndConfigSanity() above will preload the team ID map from the persisted file
-            // if it exists, so this will minimize the work that DownloadProjectTeamFiles() has to do.
-            if (REQUIRE_TEAM_WHITELIST_MEMBERSHIP)
-                DownloadProjectTeamFiles(projectWhitelist);
+            // if it exists, so this will minimize the work that DownloadProjectTeamFiles() has to do, unless explorer mode (fExplorer) is true.
+            if (REQUIRE_TEAM_WHITELIST_MEMBERSHIP || fExplorer) DownloadProjectTeamFiles(projectWhitelist);
```

Comment:
In the future, we may want to decouple `-explorer` from authorization--perhaps a new explorer may want to use the built-in scraping functionality, but we may not yet trust the entity.

Reply:
Actually what I think you are getting at here is the interaction with ALLOW_NONSCRAPER_NODE_STATS_DOWNLOAD. This is currently is not set in the appcache on mainnet (which means false) and true on testnet by appcache entry.

You are right... the net effect on mainnet is that someone cannot just act as an explorer, because they cannot download stats independently at all. On testnet, they can do it even if they are not authorized, and can also act in explorer mode.

I am not sure I want to condone the floodgates of allowing folks on mainnet to download and retain extended stats by simply setting a flag. The explorer mode downloads a lot more stuff (some of the host files are ~1 GB, and this places a higher load on the stats servers.

So for right now we will leave this alone.

Reply:
Agreed--sounds reasonable for now.